ssstar/src/create.rs

```diff
@@ -508,13 +508,18 @@ pub trait CreateProgressCallback: Sync + Send {
 
     /// An entire input object was written successfully to a tar archive.
     ///
+    /// `byte_offset` is the byte offset in the archive stream where the data for this object
+    /// starts.  Data is guaranteed to be stored in one contiguous sequence, therefore it's
+    /// possible to find the data for this object from the `byte_offset` and `size` parameters.
+    ///
     /// That doesn't mean the data written has been uploaded to remote object storage yet, it could
     /// still be buffered locally.
     fn tar_archive_object_written(
         &self,
         bucket: &str,
         key: &str,
         version_id: Option<&str>,
+        byte_offset: u64,
```

Comment:
Could you please also add timestamp `mtime: u64` here?
The value can be easily retrieved from `InputObjectPart::input_object::timestamp::timestamp`

Reply:
done